.github/workflows/test.yaml

```
@@ -0,0 +1,24 @@
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+
+    - name: Install dependencies
+      run: sudo apt-get update && sudo apt-get install -y build-essential
+
+    - name: Run tests
+      run: |
+      chmod +x ./combination_test/run_test.sh
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 23: could not find expected ':'

(syntax-check)